<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>losovátko</string>
  </property>
  <widget class="QComboBox" name="cb_year">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="zvol ročník">zvol ročník</string>
   </property>
   <item>
    <property name="text">
     <string>Vyberte třídu</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2. třída</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3. třída</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4. třída</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5. třída</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>76</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Slovíčka AJ</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_translate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>242</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Nakresli obrázek podle slovíčka</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Přelož</string>
    </property>
   </item>
  </widget>
  <widget class="QListWidget" name="lw_result">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>331</width>
     <height>187</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_start">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>80</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="te_minute">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::MinuteSection</enum>
   </property>
   <property name="displayFormat">
    <string>mm:ss</string>
   </property>
   <property name="currentSectionIndex">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
